test/system/call_list_test.rb

```diff
@@ -81,21 +77,23 @@ class CallListTest < ActionDispatch::IntegrationTest
       end
     end
 
-    it 'should time a call out after 8 hours' do
-      sign_out
-      Timecop.freeze(9.hours.from_now) do
-        log_in_as @user
-        wait_for_element 'Your completed calls'
-
-        within :css, '#completed_calls_content' do
-          assert has_no_text? @patient.name
-        end
-
-        within :css, '#call_list_content' do
-          assert has_text? @patient.name
-        end
-      end
-    end
+    # TODO flaky test and I have no idea why
```

Comment:
Do we have a followup ticket to investigate & fix this?

Reply:
Now in #1312